web/src/lib/utils/helpers.tsx

```diff
@@ -90,7 +90,7 @@ export const setPreferencesCloseSection = (preferences: Preferences, current: Se
 export const getModifiedTaskContent = (
   roadmap: Roadmap | undefined,
   task: Task,
-  field: Exclude<keyof Task, "unlockedBy">
+  field: Exclude<keyof Task, "unlockedBy" | "required">
```

Comment:
Ah, since it only returned `string`, I was thinking the helper is used to get displayable content like the name and CTA text. Not sure if returning `string | boolean` will fit into the helper's design here, as all the callers are expecting a `string` back to feed into components like `<Content />` or `<TaskCTA />`.

Reply:
Ah I see your point